packages/liquidsoap-daemon/liquidsoap-daemon.0.3.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "romain.beauxis@gmail.com"
+homepage: "https://github.com/savonet/liquidsoap-daemon"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+available: [ os != "win32" ]
+install: [
+  ["./daemonize-liquidsoap.sh"] { os = "linux" }
```

Comment:
Looking at the script, it appears to call sudo for removal as well

```
            sudo systemctl disable liquidsoap
            sudo systemctl stop liquidsoap
            sudo rm "$systemd_target"
            sudo systemctl daemon-reload
```

I think any invocation of sudo like this as part of the opam install would be the problem -- we don't really have a good way to hook into system daemon scripts in a portable way, so the best way is to print a `messages:` field asking the user to run the script manually via sudo.  @altgr may have a better idea, so CCIng him